packages/fiori/src/SideNavigationItem.ts

```diff
@@ -201,6 +225,12 @@ class SideNavigationItem extends SideNavigationSelectableItemBase {
 	get isSideNavigationItem() {
 		return true;
 	}
+
+	_toggle() {
```

Comment:
In ui5-side-navigation, in collapsed mode - go to a single click area parent, open the menu with children - press enter or space on the first item (this would be the parent itself) - menu closes. Next time when you open the menu - the child items are gone

Reply:
Added as leftover for another change. We need more info on how unselectable parents should behave in collapsed mode.